WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -1046,7 +1046,7 @@ extension GutenbergViewController: GutenbergBridgeDataSource {
             .contactInfoBlock: post.blog.supports(.contactInfo) && FeatureFlag.contactInfo.enabled,
             .unsupportedBlockEditor: isUnsupportedBlockEditorEnabled,
             .canEnableUnsupportedBlockEditor: post.blog.jetpack?.isConnected ?? false,
-            .audioBlock: !isFreeWPCom, // Disable audio block until it's usable on free sites via "Insert from URL" capability
+            .audioBlock: !isFreeWPCom,
```

Comment:
Related to this [comment](https://github.com/WordPress/gutenberg/pull/31966#discussion_r635667170), what do you think about renaming the capability?

Reply:
Done here 2d20967